exec/csnex/Number.cs

```diff
@@ -23,9 +23,34 @@ public static Number FromString(string str)
             return new Number(Decimal.Parse(str, System.Globalization.NumberStyles.Float));
         }
 
+#region Static Arithmetic Functions
+        public static Number Subtract(Number x, Number y)
+        {
+            return new Number(x.val - y.val);
+        }
+#endregion
+
+        public bool IsInteger()
+        {
+            Decimal i = Decimal.Round(val, MidpointRounding.AwayFromZero);
```

Comment:
Why do this weird rounding mode? Truncating the number would be more appropriate here.

Reply:
Changed to truncate `val`.